src/stories/Library/banner/Banner.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC, ReactNode } from "react";
+import clsx from "clsx";
+import MediaContainer from "../media-container/MediaContainer";
+import { ReactComponent as ArrowLargeRight } from "../Arrows/icon-arrow-ui/icon-arrow-ui-large-right.svg";
+
+type BannerType = {
+  title: string;
+  image?: ReactNode;
+  description?: string;
+};
```

Comment:
In the case that a link is required, i think this is missing a required link as prop.